config_example.json

```diff
@@ -25,6 +25,16 @@
     "scopes": ["project:releng:googleplay:aurora"],
     "google_play_track": "alpha"
   },
-  "repository_to_watch": "releases/mozilla-aurora",
-  "watch_interval_in_seconds": 60
+  "pulse": {
+    "host": "pulse.mozilla.org",
+    "port": "5671",
+    "user": "YOUR_USER",
+    "password": "YOUR_PASSWORD",
+    "queue": "YOUR_QUEUE_NAME",
+    "exchanges": [{
+      "path": "exchange/taskcluster-queue/v1/task-completed",
+      "routing_keys": ["route.index.gecko.v2.mozilla-aurora.nightly.latest.mobile.#"]
+    }]
+  },
+  "verbose": true
```

Comment:
This file looks like a duplicate of the default config file. I bet we'll be forgetting to update both in the future. :) How about deleting this file and using the default config file as an example config instead (in the docs).